yash.md

```diff
@@ -2,7 +2,85 @@
 
 _A combination index, glossary, and high level guided tour for the Stanford Digital Repository_
 
-* Access Rights for digital content
+Intended Audience: infrastructure team developers
+
+## Argo
```

Comment:
Duplicative of https://github.com/sul-dlss/infrastructure-team/blob/gh-pages/sdr/index.md#argo